btcec/schnorr/musig2/nonces.go

```diff
@@ -22,6 +19,10 @@ const (
 	SecNonceSize = 64
 )
 
+// zeroSecNonce is a secret nonce that's all zeroes. This is used to check that
+// we're not attempting to re-use a nonce, and also protect callers from it.
+var zeroSecNonce [SecNonceSize]byte
```

Comment:
unused?

Reply:
Ah yeah, was left over from an attempt to zero out nonces when signing, but ended up rolling that context struct in stead.